catroid/src/main/java/org/catrobat/catroid/content/Script.java

```diff
@@ -258,4 +258,15 @@ public void setCommentedOut(boolean commentedOut) {
 			}
 		}
 	}
+
+	public ArrayList<Brick> getBricksRequiringResources(int resource) {
+		ArrayList<Brick> resourceBrickList = new ArrayList<Brick>();
```

Comment:
note: only on the left side of the "=". When creating it with "new" you have to specify the Implementation.

see: http://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java